build.sbt

```diff
@@ -134,6 +140,7 @@ lazy val commonSettings = Seq(
   resolvers ++= Seq(Resolver.sonatypeRepo("releases"), Resolver.sonatypeRepo("snapshots")),
   parallelExecution in Test := false,
   testFrameworks += new TestFramework("munit.Framework"),
+  doctestTestFramework := DoctestTestFramework.Munit,
```

Comment:
Why is this necessary?

Reply:
It isn't. Can remove.

Although isn't a bad idea to explicitly set `doctestTestFramework`, and ensure relevant test library dependencies are available. Since MUnit dependencies are explicitly provided in the build for most modules, set `doctestTestFramework` to that, but can be set to any supported framework.

Reply:
Removed.